### ISSUE TITLE
Some parameters are not being copied from JSON

### ISSUE THREAD

**Issue by wicked-head** [TheLastGimbus/GooglePhotosTakeoutHelper#195]
Hi, first let me thank you for your dedication and sharing this tool to the world!

I performed a test by manually removing some tags from the file even if the information was already in the json file and it did not did copy everything. 
Example:
   Step 1) Right click on image -> Properties -> Details -> Remove Properties and Personal Information -> deleted "Date Taken", "GPS information", Title, "subject"
   Step 2) ran the gpth-windoza.exe with admin rights
   Step 3) target photo only changed the "Last Modified Date" is now same date as in the JSON "photoTakenTime" tag. 
             **But it is Missing:** "Date Taken", "GPS" information (Lat, Long, Alt), Title and Description tags could also be copied. This information is present on the JSON file but was not on the original file metadata as part of my test, and my expectation was that (at least) the GPS information would been enriched by your program.

Additionally, shouldnt it also change the "Date Created" tag? or is that not accessible to overwrite?

Is it something that could be easily done as you already have the framework to do it?

Thanks for all the good work :)

**Comment by bartaspoz** [comment]
Hey! Thanks for your awesome tools. In the readme there is an info takt moving GPS data from json to exif is working for python version. So could you tell it we would like to get the GPS data from json we need to use the 2x version?
If yes, I have another question. Am I able to re-run the python version on data alredy parsed by most recent version?

I have now input folder with all Json files, and output folder, nicely arranged by dates, but withiut json files.

**Comment by TheLastGimbus** [comment]
> is it possible to re-run the python version on top of already alredy parsed data by

I don't think so, and I don't recommend touching python version at all - it was a mess, and I can't tell you for sure what will happen

I think the fastest would be to write a new 20-line python script that would do that  - tho it wouldn't have any advanced json matching like whole gpth has

...or, I could add experimental feature to gpth to generate a exiftool-based bash script to set all of those locations

what do you think?

**Comment by bartaspoz** [comment]
Hi, any way of preserving gps data would be a gamechanger. I use photos location history constantly, as I travel a lot and like to re-visit places by searching for them on the map. My idea is to import Goole Takeout to Synology Photos, where the map feature is present as well. 

So I tkink your last idea is the one that I like the most :) 
I would love to help you, but I'm not a programmer.

**Comment by TheLastGimbus** [comment]
Wait, but... as I tried to reaserch, as in #191 , GPhotos doesn't strip exifs - I don't remember *any* precisely observed case of this - so, did you actaully check if your photos in your takeout have exifs? (Exif is metadata format that contains the location info). If they do have it, you don't need to re-merge that fom jsons...

If *all* of your photos have this stripped, then i want to investigate this very intense - did you have an iphone, or did most of your photos *did not* have gps data from camera, but you *manually edited* that in gphotos app? @bartaspoz

**Comment by alexander-myltsev** [comment]
Hi. Thanks for your software.

I'm exporting ~0,5Tb of photos / videos. There are thousands of photos with missing geolocation (checked at Google Photos) and without date after processing with `gpth`. From where should I start to trace reasons why there is no such metadata?

**Comment by TheLastGimbus** [comment]
@alexander-myltsev did your camera even collect it? is it visible in the app? did you change it manually? etc etc

**Comment by alexander-myltsev** [comment]
> did your camera even collect it? is it visible in the app? 

Both yes. There is that metadata in Google Photos.

> did you change it manually? etc etc

Definitely no.

**Comment by matthewmcneill** [comment]
Just some additional info here - based mainly on researching how to use EXIFTool by Phil Harvey.  Perhaps add these to the readme for now?

Discussion on Phil Harvey's site:
https://exiftool.org/forum/index.php?topic=12361.0

Pretty comprehensive summary to cope with -edited.* and json files and also options to process ALL tags in the json.
https://github.com/m1rkwood/google-photos-takeout-scripts/blob/main/README.md

M
